zerver/lib/import_realm.py

```diff
@@ -219,11 +219,17 @@ def fix_message_rendered_content(data: TableData, field: TableName) -> None:
             continue
 
         try:
-            rendered_content = save_message_rendered_content(message_object, message['content'])  # type: Optional[str]
+            content = message['content']
+            realm = message_object.get_realm()
+            rendered_content = render_markdown(
+                message=message_object,
+                content=content,
+                realm=realm,
+            )
+            message_object.rendered_content = rendered_content
+            message_object.rendered_content_version = bugdown_version
+            message_object.save_rendered_content()
         except Exception:
-            rendered_content = None
-
-        if rendered_content is None:
```

Comment:
I think removing this block breaks things; check the code.  The problem is that `bugdown` itself returns `None` in the event of e.g. a timeout.

Reply:
So the reason I removed those was that the  `mypy` annotations in the stack were all `str` instead of `Optional[str]`, and it appears that we throw exceptions for error cases, although the `timeout` code is a bit unclear on that.  Inlining this function and keeping `render_content` for some reason forces me into a pretty large cascade of `mypy` changes.

Reply:
So I ended up doing the `assert(rendered_content is not None)` hack.